zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -640,6 +653,13 @@ private boolean canScanNode(StructuralNode node) {
             }
             return false;
         }
+        Set<FilterResult> filterResult = filterNode(node);
```

Comment:
I have not thought about how and where to indicate to user the reason for filtering.

Reply:
Other scans show a tab/table with the reason and details of the message, we could do the same here.